### PR TITLE
lastMessage in chat

### PR DESCRIPTION
**Pull request by tofers**
I need to get the last message from the dialog. How is it done in the official WhatsApp Web.

### CODE REVIEW

**Comment by PurpShell** [comment]
you can use fetchMessage for that (it is sorted by the newest 50 messages and back)

**Review by tuyuribr — CHANGES_REQUESTED** [review]
index.d.ts docs

File: index.d.ts [hunk 1]
```diff
@@ -1018,6 +1018,8 @@ declare namespace WAWebJS {
         timestamp: number,
         /** Amount of messages unread */
         unreadCount: number,
+        /** Last message fo chat */
```
Comment (tuyuribr) [inline index.d.ts:1021]: typo

**Review by tuyuribr — CHANGES_REQUESTED** [review]
LGTM

**Review by tofers — COMMENTED** [review]
What is wrong in the description?

File: src/util/Injected.js [hunk 2]
```diff
@@ -400,7 +400,15 @@ exports.LoadUtils = () => {
             await window.Store.GroupMetadata.update(chatWid);
             res.groupMetadata = chat.groupMetadata.serialize();
         }
-
+        
+        res.lastMessage = null;
+        if (res.msgs && res.msgs.length) {
+            const lastMessage = window.Store.Msg.get(chat.lastReceivedKey._serialized);
```
Comment (PurpShell) [inline src/util/Injected.js:406]:
```suggestion
            const lastMessage = window.Store.Msg.find(a => a.id._serialized == chat.lastReceivedKey._serialized);
```

Comment (PurpShell) [reply src/util/Injected.js:406]: .get is sometimes unreliable

**Review by purpshell — CHANGES_REQUESTED** [review]

File: src/util/Injected.js (hunk 2 above)
Comment (tofers) [reply src/util/Injected.js]: this doesn't work for me, causes message serialization error

Comment (PurpShell) [reply src/util/Injected.js:426]: Now?

Comment (tofers) [reply src/util/Injected.js:426]: it works stably

**Review by tuyuribr — APPROVED** [review]

**Review by purpshell — APPROVED** [review]
Looks good

**Pull request merged**